src/pages/HomePage/HomePage.tsx

```diff
@@ -56,7 +56,7 @@ const HomePage: React.FC<HomePageProps> = ({ isLoaded, withGamification }) => {
       .then((res) => res.json())
       .catch((err) => console.error("Error fetching user counts ", err));
 
-    if (roomsWithUserCount !== undefined) {
+    if (roomsWithUserCount.length > 0) {
```

Comment:
what if roomsWithUserCount is undefined ?

Reply:
I think then the execution of this function will fail, and default value will be used.

Reply:
would be nice to handle this case in the code, otherwise ok

Reply:
Added check for that.